ocaml/idl/ocaml_backend/gen_api.ml

```diff
@@ -28,7 +28,7 @@ let overrides = [
     "let rpc_of_vm_operations_to_string_map x = Rpc.Dict (List.map (fun (x,y) -> (match rpc_of_vm_operations x with Rpc.String x -> x | _ -> failwith \"Marshalling error\"), Rpc.String y) x)\n" ^
     "let vm_operations_to_string_map_of_rpc x = match x with Rpc.Dict l -> List.map (function (x,y) -> vm_operations_of_rpc (Rpc.String x), string_of_rpc y) l | _ -> failwith \"Unmarshalling error\"\n");
   "bond_mode",(
-    "let rpc_of_bond_mode x = match x with `balanceslb -> Rpc.String \"balance-slb\" | `activebackup -> Rpc.String \"active-backup\" | `lacp -> Rpc.String \"lacp\"\n"^
+    "let rpc_of_bond_mode x = match x with `balanceslb -> Rpc.String \"balance-slb\" | `activebackup -> Rpc.String \"active-backup\" | `lacp -> Rpc.String \"lacp\" | `unknown -> failwith \"No marshalling `unknown values\" \n"^
     "let bond_mode_of_rpc x = match x with Rpc.String \"balance-slb\" -> `balanceslb | Rpc.String \"active-backup\" -> `activebackup | Rpc.String \"lacp\" -> `lacp | _ -> failwith \"Unmarshalling error in bond-mode\"\n");
```

Comment:
In the `xyz_of_rpc` overrides, should the `match` blocks end with
```_ -> `unknown``` rather than
`_ -> Failwith "..."`
?

Reply:
Ah yes, good spot - we should have the same logic there as is in ocaml-rpc (ie, it matches String _ and Enum [String _ :: _])